pkgs/development/libraries/glib/default.nix

```diff
@@ -57,7 +57,7 @@ stdenv.mkDerivation rec {
     ++ optional doCheck ./skip-timer-test.patch
     ++ [ ./schema-override-variable.patch ];
 
-  outputs = [ "out" "dev" "devdoc" ];
+  outputs = [ "bin" "out" "dev" "devdoc" ];
```

Comment:
Though I am not sure if the `bin` output should be the primary one. cc @vcunat

Reply:
Yes that's correct. The intention is anybody can write `${pkgs.foo}/bin/bar` in their systemd unit scripts or whatever and have it work.

Reply:
Probably.  So far we've only broken the convention for `glibc` AFAIK.

Reply:
but beware of all the code using `${glib}/lib/` and similar paths